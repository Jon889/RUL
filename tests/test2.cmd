./RUL hsr2.txt print 0x0d001a00,0,0 0x0d031500,0,0
