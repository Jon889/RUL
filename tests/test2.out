
      0,0     0,0
   +-------+-------+   +-------+-------+
   |   0   |   2   |   |   0   |   2   |
N1:|0     1|0     0| O:|0     1|0     0|
   |   3   |   2   |   |   3   |   2   |
   +-------+-------+   +-------+-------+
    (null)  (null) 
